modules/cpdos/hbh.py

```diff
@@ -28,10 +28,10 @@ def CachePoisoning(url, s, params2, resp1, resp2, authent, headers):
         pass
         #print(f"Error : {e}")
     
-    if resp3.status_code == resp2.status_code and resp3.status_code != resp1.status_code and resp2.status_code != 429:
+    if resp3.status_code == resp2.status_code and resp3.status_code != resp1.status_code and resp3.status_code != 429:
```

Comment:
At this point resp2 status code is the same as resp3 status code, just making sure that the last request done (resp3) isn't tainted with a 429